### ISSUE TITLE
BO - Currencies - It is not possible to create a BYN currency

### ISSUE THREAD

**Issue by khouloudbelguith** [PrestaShop/PrestaShop#24692]
#### Describe the bug

In the BO > International > Localization > Currencies page, the we try to create a BYN currency => an exception is displayed

#### Steps to Reproduce

1. Go to BO > International > Localization > Currencies page
2. Click on 'Add new currency'
3. Choose BYN

![image](https://user-images.githubusercontent.com/16067358/119974963-98e40c00-bfb5-11eb-99ad-0b3927bc65e8.png)

In the PS1769 => ok
![image](https://user-images.githubusercontent.com/16067358/119975043-b022f980-bfb5-11eb-90f4-319fd40f5de8.png)

#### Additional information

* PrestaShop version: 177x (1775)
* PHP version: 7.2

**Comment by aespasa** [comment]
It looks like the numericIsoCode is missing for few currency data, but I can't figure out where he is generated. I saw that Prestashop is using CLDR data, they are stored in the localization/core/common/main but I can't figure out where the numeric iso code is managed..

This may help someone else...

All currencies without numericIsoCode:

```
PrestaShop\PrestaShop\Core\Localization\CLDR\CurrencyData Object
(
    [isoCode:protected] => BYN
    [numericIsoCode:protected] => 
    [decimalDigits:protected] => 2
    [displayNames:protected] => 
    [symbols:protected] => Array
        (
            [narrow] => р.
        )

    [active:protected] => 1
)
PrestaShop\PrestaShop\Core\Localization\CLDR\CurrencyData Object
(
    [isoCode:protected] => ESP
    [numericIsoCode:protected] => 
    [decimalDigits:protected] => 0
    [displayNames:protected] => 
    [symbols:protected] => Array
        (
            [narrow] => ₧
        )

    [active:protected] => 
)
PrestaShop\PrestaShop\Core\Localization\CLDR\CurrencyData Object
(
    [isoCode:protected] => LVL
    [numericIsoCode:protected] => 
    [decimalDigits:protected] => 2
    [displayNames:protected] => 
    [symbols:protected] => Array
        (
            [narrow] => Ls
        )

    [active:protected] => 
)
PrestaShop\PrestaShop\Core\Localization\CLDR\CurrencyData Object
(
    [isoCode:protected] => RUR
    [numericIsoCode:protected] => 
    [decimalDigits:protected] => 2
    [displayNames:protected] => 
    [symbols:protected] => Array
        (
            [narrow] => р.
        )

    [active:protected] => 
)
```

**Comment by olecorre** [comment]
Is in Localisation/CDLR/core/common/supplemental/supplementalData.xml line 5148 add a line with `<currencyCodes type="BYN" numeric="933"/>`

**Comment by olecorre** [comment]
I create a PR https://github.com/PrestaShop/PrestaShop/pull/24763

**Comment by Progi1984** [comment]
Fixed by @olecorre in #24770
